1.18 version posted to CurseForge requires 1.17.x

net.fabricmc.loader.impl.FormattedException: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'End Goblin Traders' (endgoblintraders) 1.4.0 (G:\Games\Minecraft\MultiMC\instances\Bunny Slippers Test\.minecraft\mods\end-goblin-traders-1.4.0.jar) with any version that is compatible with:
		 - minecraft 1.18
Unmet dependency listing:
	 - Mod 'End Goblin Traders' (endgoblintraders) 1.4.0 requires any 1.17.x version of 'Minecraft' (minecraft), but only the wrong version is present: 1.18!
		 - 'Minecraft' (minecraft) 1.18 is an environment reference and usually requires installation or launcher changes to adjust
Inactive mods:
	 - mod 'End Goblin Traders' (endgoblintraders) 1.4.0, reason: to replace
	 - mod 'Cloth Basic Math' (cloth-basic-math) 0.5.1, reason: newer version active
	 - mod 'Cloth Config v6' (cloth-config) 6.0.42, reason: same version active
	 - mod 'Fabric API Base' (fabric-api-base) 0.1.3+12a8474c34, reason: newer version active
	 - mod 'Fabric API Base' (fabric-api-base) 0.3.0+a02b446388, reason: newer version active
	 - mod 'Fabric API Base' (fabric-api-base) 0.4.0+65d505fc9c, reason: newer version active
	 - mod 'Fabric Command API (v1)' (fabric-command-api-v1) 1.1.2+6cefd57718, reason: newer version active
	 - mod 'Fabric Networking (v0)' (fabric-networking-v0) 0.3.3+65d505fc9c, reason: newer version active
	 - mod 'Fabric Resource Loader (v0)' (fabric-resource-loader-v0) 0.4.7+b7ab612118, reason: newer version active
	 - mod 'Fabric Resource Loader (v0)' (fabric-resource-loader-v0) 0.4.8+a00e834b88, reason: newer version active
	 - mod 'Fabric Screen API (v1)' (fabric-screen-api-v1) 1.0.4+cbda931888, reason: newer version active
	 - mod 'Jankson' (jankson) 3.0.0+j1.2.0, reason: newer version active
	 - mod 'LibGui' (libgui) 5.0.0-beta.1+1.18-pre2, reason: newer version active
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:188)
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:142)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:71)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: net.fabricmc.loader.impl.discovery.ModResolutionException: Mod resolution encountered an incompatible mod set!
A potential solution has been determined:
	 - Replace mod 'End Goblin Traders' (endgoblintraders) 1.4.0 (G:\Games\Minecraft\MultiMC\instances\Bunny Slippers Test\.minecraft\mods\end-goblin-traders-1.4.0.jar) with any version that is compatible with:
		 - minecraft 1.18
Unmet dependency listing:
	 - Mod 'End Goblin Traders' (endgoblintraders) 1.4.0 requires any 1.17.x version of 'Minecraft' (minecraft), but only the wrong version is present: 1.18!
		 - 'Minecraft' (minecraft) 1.18 is an environment reference and usually requires installation or launcher changes to adjust
Inactive mods:
	 - mod 'End Goblin Traders' (endgoblintraders) 1.4.0, reason: to replace
	 - mod 'Cloth Basic Math' (cloth-basic-math) 0.5.1, reason: newer version active
	 - mod 'Cloth Config v6' (cloth-config) 6.0.42, reason: same version active
	 - mod 'Fabric API Base' (fabric-api-base) 0.1.3+12a8474c34, reason: newer version active
	 - mod 'Fabric API Base' (fabric-api-base) 0.3.0+a02b446388, reason: newer version active
	 - mod 'Fabric API Base' (fabric-api-base) 0.4.0+65d505fc9c, reason: newer version active
	 - mod 'Fabric Command API (v1)' (fabric-command-api-v1) 1.1.2+6cefd57718, reason: newer version active
	 - mod 'Fabric Networking (v0)' (fabric-networking-v0) 0.3.3+65d505fc9c, reason: newer version active
	 - mod 'Fabric Resource Loader (v0)' (fabric-resource-loader-v0) 0.4.7+b7ab612118, reason: newer version active
	 - mod 'Fabric Resource Loader (v0)' (fabric-resource-loader-v0) 0.4.8+a00e834b88, reason: newer version active
	 - mod 'Fabric Screen API (v1)' (fabric-screen-api-v1) 1.0.4+cbda931888, reason: newer version active
	 - mod 'Jankson' (jankson) 3.0.0+j1.2.0, reason: newer version active
	 - mod 'LibGui' (libgui) 5.0.0-beta.1+1.18-pre2, reason: newer version active
	at net.fabricmc.loader.impl.discovery.ModResolver.findCompatibleSet(ModResolver.java:183)
	at net.fabricmc.loader.impl.discovery.ModResolver.resolve(ModResolver.java:48)
	at net.fabricmc.loader.impl.FabricLoaderImpl.setup(FabricLoaderImpl.java:222)
	at net.fabricmc.loader.impl.FabricLoaderImpl.load(FabricLoaderImpl.java:186)
	... 11 more

--- a/gradle.properties
+++ b/gradle.properties
@@ -8,7 +8,7 @@ org.gradle.jvmargs=-Xmx1G
 	loader_version=0.12.1
 
 # Mod Properties
-	mod_version = 1.4.0
+	mod_version = 1.4.1
 	maven_group = com.jab125
 	archives_base_name = end-goblin-traders
 

--- a/src/main/resources/fabric.mod.json
+++ b/src/main/resources/fabric.mod.json
@@ -38,8 +38,8 @@
     "fabricloader": ">=0.11.3",
     "fabric": "*",
     "minecraft": ">=1.17",
-    "goblintraders": ">=1.3.0",
-    "thonkutil": ">=1.2",
+    "goblintraders": ">=1.4.0",
+    "thonkutil": ">=1.3",
     "cloth-config2": ">=5.0.38",
     "java": ">=16"
   },

--- a/src/main/resources/fabric.mod.json
+++ b/src/main/resources/fabric.mod.json
@@ -37,7 +37,7 @@
   "depends": {
     "fabricloader": ">=0.11.3",
     "fabric": "*",
-    "minecraft": "1.17.x",
+    "minecraft": ">=1.17",
     "goblintraders": ">=1.3.0",
     "thonkutil": ">=1.2",
     "cloth-config2": ">=5.0.38",

